test/e2e/network/network_policy.go

```diff
@@ -85,8 +90,29 @@ var _ = SIGDescribe("NetworkPolicy", func() {
 			testCannotConnect(f, f.Namespace, "client-cannot-connect", service, 80)
 		})
 
-		ginkgo.It("should enforce policy based on PodSelector [Feature:NetworkPolicy]", func() {
-			ginkgo.By("Creating a network policy for the server which allows traffic from the pod 'client-a'.")
+		ginkgo.It("should enforce policy to allow traffic from pods within server namespace based on PodSelector [Feature:NetworkPolicy]", func() {
+			nsA := f.Namespace
+			nsBName := f.BaseName + "-b"
+			// The CreateNamespace helper uses the input name as a Name Generator, so the namespace itself
+			// will have a different name than what we are setting as the value of ns-name.
+			// This is fine as long as we don't try to match the label as nsB.Name in our policy.
```

Comment:
I don't think this comment is necessary (in any of the places where it appears).

Reply:
I have removed the comments from all the places in the file.